apollo-router-core/src/layers/deduplication.rs

```diff
@@ -1,130 +0,0 @@
-use crate::{fetch::OperationKind, http_compat, Request, SubgraphRequest, SubgraphResponse};
-use futures::{future::BoxFuture, lock::Mutex};
```

Comment:
Are we deleting the de-duplication layer because we always want de-duplication and we always want it to be merged with caching (which we also always want)? I'm assuming that is the case, but just making sure.

Reply:
I am hoping that this caching layer will provide enough real world utility to address both needs. It's caching with dedup, ttl and tti.
Not ruling out that we may need a dedicated dedup layer down the line that has the same behaviour as before.